### PR TITLE
breaking: Remove UserSettableParameter

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

**Review by Corvince — APPROVED** [review]

File: tests/test_visualization.py [hunk 1]
```diff
@@ -6,7 +6,10 @@
 from mesa.time import SimultaneousActivation
 from mesa.visualization.ModularVisualization import ModularServer
 from mesa.visualization.modules import CanvasGrid, TextElement
-from mesa.visualization.UserParam import UserSettableParameter
+from mesa.visualization.UserParam import (
+    NumberInput,
+    Slider,
+)
 from tests.test_batchrunner import MockAgent
```
Comment (tpike3) [inline tests/test_visualization.py:13]: I think I am missing some here on the conflict. If we are importing `MockAgent` shouldn't it be `tests.test_batch_run`?

Comment (rht) [reply tests/test_visualization.py]: That was the code from before `BatchRunner` was removed.

**Comment by rht** [comment]
Merge conflict resolved.

**Review by tpike3 — APPROVED** [review]
LGTM

**Pull request merged**